201-vm-linux-jupyterhub/azuredeploy.json

```diff
@@ -325,6 +356,12 @@
                     "computerName": "[parameters('vmName')]",
                     "adminUsername": "[parameters('adminUsername')]",
                     "adminPassword": "[parameters('adminPassword')]"
+                },
+                "diagnosticsProfile": {
+                    "bootDiagnostics": {
+                        "enabled": true,
+                        "storageUri": "[concat('https://', variables('storageAccountName'), '.blob.core.windows.net/')]"
```

Comment:
use reference() to get the uri

Reply:
bringing alignment to 101-vm-linux-jupyerhub template but having larger machine sizes